src/uct/ib/dc/dc_mlx5.c

```diff
@@ -540,36 +625,82 @@ uct_dc_mlx5_iface_ooo_flag(uct_dc_mlx5_iface_t *iface, uint64_t flag,
 }
 #endif
 
+ucs_status_t uct_ib_mlx5_set_srq_dc_params(uct_dc_mlx5_iface_t *iface)
```

Comment:
maybe `uct_dc_mlx5_set_srq_params`, since it is not relevant for other IB transports?

Reply:
uct_ib_mlx5_devx_set_srq_dc_params